wremnants/syst_tools.py

```diff
@@ -219,11 +220,19 @@ def define_mass_weights(df, proc):
 
     return df
 
-def add_massweights_hist(results, df, axes, cols, base_name="nominal", proc=""):
+def add_massweights_hist(results, df, axes, cols, base_name="nominal", proc="", addhelicity=False):
     name = Datagroups.histName(base_name, syst="massWeight"+(proc[0] if len(proc) else proc))
-    massWeight = df.HistoBoost(name, axes, [*cols, "massWeight_tensor_wnom"], 
-                    tensor_axes=[hist.axis.StrCategory(massWeightNames(proc=proc), name="massShift")], 
-                    storage=hist.storage.Double())
+    mass_axis = hist.axis.StrCategory(massWeightNames(proc=proc), name="massShift")
+    if addhelicity:
+        massweightHelicity, massWeight_axes = make_massweight_helper_helicity(mass_axis)
```

Comment:
The massweight tensor for the non helicity case is defined in the ```define_mass_weights``` function, can we make this consistent, e.g. move these lines into ```define_mass_weights``` and make the case distinction between helicity or non helicity there?